content/en/docs/concepts/services-networking/endpointslice.md

```diff
@@ -0,0 +1,95 @@
+---
+reviewers:
+- freehan
+title: Endpoint Slice
+feature:
+  title: Endpoint Slices
+  description: >
+    Scalable tracking of network endpoints in a Kubernetes cluster.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< glossary_definition term_id="endpointslice" length="short" >}}
+
+_Endpoint Slices_ provide a simple way to track network endpoints within a
+Kubernetes cluster. They offer a more scalable alternative to Endpoints
+resources within Kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Endpoint Slice resources {#endpointslice-resource}
+
+In Kubernetes, an Endpoint Slice contains references to a set of network
+endpoints. The EndpointSlice controller automatically creates Endpoint Slices
+for a Kubernetes Service with a selector is specified. These Endpoint Slices
```

Comment:
nit: line 21, `Endpoints resources` vs `Endpoint resources` vs. `Endpoints`. You can probably drop resource 
nit: for a Kubernetes Service when a selector ... ?

Reply:
Thanks! All this terminology is confusing. Trying to differentiate Kubernetes Endpoints resources from the generic concept of network endpoints is hard to balance, hopefully it's all reasonably clear now, but let me know if there's a better way to communicate the difference here.